paper/src/main/java/com/denizenscript/denizen/paper/properties/PaperEntityExtensions.java

```diff
@@ -184,6 +186,40 @@ public static void register() {
                 }
                 object.getLivingEntity().damageItemStack(slot.asEnum(EquipmentSlot.class), amount.asInt());
             });
+
+            // <--[mechanism]
+            // @object EntityTag
+            // @name shear
+            // @Plugin paper
+            // @group paper
+            // @description
+            // Shears a sheep, harvests a bogged or mushroom cow or derps a snow golem as a player would do, including drops.
+            // If the entity is not ready to be shared, there are no drops but the sound will still play.
+            // Optionally, specify a sound source to change the source of the sound.
+            // Valid types are AMBIENT, BLOCKS, HOSTILE, MASTER, MUSIC, NEUTRAL, PLAYERS, RECORDS, VOICE and WEATHER

```

Comment:
I'd link to the JD page for sound sources, saves us having to manually update it in the future.